packages/frontend/src/scripts/sound.ts

```diff
@@ -125,9 +125,16 @@ export async function loadAudio(url: string, options?: { useCache?: boolean; })
 export function playMisskeySfx(operationType: OperationType) {
 	const sound = defaultStore.state[`sound_${operationType}`];
 	if (sound.type == null || !canPlay || ('userActivation' in navigator && !navigator.userActivation.hasBeenActive)) return;
+	playMisskeySfxFile(sound);
+}
+
+export async function playMisskeySfxFile(soundStore: SoundStore) {
+	if (soundStore.type === null || (soundStore.type === '_driveFile_' && !soundStore.fileUrl)) {
```

Comment:
ここに、127行目のように`!canPlay`を追加しないといけないような気がします…?
```suggestion
	if (soundStore.type === null || !canPlay || (soundStore.type === '_driveFile_' && !soundStore.fileUrl)) {
```